src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import './StudentCoordinatorSettings.css';
+import OrganizationDetails from './OrganizationDetails';
+import AccountManager from './AccountManager';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            account: null,
+            org: null,
+            coordinator: null,
+            secondaryContact: null,
+            isEditingOrg: false,
+            isEditingAcc: false,
+            email: 'phisigmaro@uw.edu'
+        };
+    }
+
+    // Backend TODO: Get data from DB
+    componentDidMount() {
+        let account  = {
```

Comment:
You actually shouldn't mock any data, we're already grabbing `account` data of the logged in user, so use `props.account`. This way you can also actually log in with different users and test.

Reply:
Fixed.